src/courseware/data/api.js

```diff
@@ -229,3 +230,65 @@ export async function getCourseTopics(courseId) {
     .get(`${getConfig().LMS_BASE_URL}/api/discussion/v2/course_topics/${courseId}`);
   return camelCaseObject(data);
 }
+
+/**
+ * Normalizes outline blocks for a given course.
+ * @param {string} courseId - The unique identifier for the course.
+ * @param {Object} blocks - An object containing different blocks of the course outline.
+ * @returns {Object} - An object with normalized sections, sequences, and units.
+ */
+export function normalizeOutlineBlocks(courseId, blocks) {
```

Comment:
Maybe it should be moved to `utils.js`?

Reply:
In this file we have the same implementation because of that I didn't move to utils.js.
But moving it to the separate file might be better.
I'll do a small refactoring of this file.